samples/ipc/ipc_service/Kconfig.sysbuild

```diff
@@ -8,3 +8,6 @@ source "${ZEPHYR_BASE}/share/sysbuild/Kconfig"
 
 config REMOTE_BOARD
 	string "The board used for remote target"
+	default "nrf5340dk/nrf5340/cpunet" if BOARD_NRF5340DK_NRF5340_CPUAPP
+	default "nrf5340dk/nrf5340/cpunet" if BOARD_NRF5340DK_NRF5340_CPUAPP_NS
+	default "nrf54h20dk/nrf54h20/cpurad" if BOARD_NRF54H20DK_NRF54H20_CPUAPP
```

Comment:
Can we also add default configuration for PPR?
Then the sysbuild_cpuppr.conf will be unnecessary.

Reply:
Not really. Default target for nRF54h20 cpuapp is cpurad